src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -92,8 +92,9 @@ private sealed partial class UIThreadOnly : IDisposable
 
             private readonly IWaitIndicator _waitIndicator;
 
-            public readonly ITextBuffer OutputBuffer;
-            public readonly ITextBuffer StandardInputBuffer;
+            public ITextBuffer OutputBuffer { get; }
+            public ITextBuffer HistoryBuffer { get; private set; }
```

Comment:
OK, but we could use the same underlying primitives as AddInput does instead of adding a new buffer, right? I don't like introducing a new buffer since language services would now need to understand it.

---
In reply to: [62404892](https://github.com/dotnet/roslyn/pull/11078#discussion_r62404892) [](ancestors = 62404892)